app/src/components/containers/attributeListContainer/attributeListContainer.jsx

```diff
@@ -22,7 +22,7 @@ import { EditableAttributeList } from 'componentLibrary/attributeList/editableAt
 export const AttributeListContainer = ({ value, keyURLCreator, valueURLCreator, ...rest }) => {
   const projectId = useSelector(activeProjectSelector);
   const getURIKey = keyURLCreator(projectId);
-  const getURIValue = (key) => keyURLCreator(projectId, key);
+  const getURIValue = valueURLCreator(projectId);
```

Comment:
Same question about the value search here.
Do we really need to remove that feature?